### DISCUSSION TITLE
ImageMagick for converting Photos on a NAS

### DISCUSSION THREAD

**Discussion by Gothmoth88** [ImageMagick/ImageMagick#2405] [Help]
hello,

how can i use imagemagick to convert photos (tiff format to jpg) on a nas?

my goal is to automatically convert all new TIFF files that i backup on my NAS to JPG and store the JPG files in a second folder on the NAS.
qnap lets me install imagmagick 8 but i have no idea how to do that. i have never touched imagmagick before.

thx.

**Comment by fmw42** [comment]
There is no ImageMagick 8, only 6 and 7.

What is your platform/OS?

See how to install binaries for each platform at https://imagemagick.org/script/download.php

**Comment by Gothmoth88** [comment]
hi,

it is called imagemagick q8 on the qnapclub server

https://www.qnapclub.eu/de/qpkg/586


> See how to install binaries for each platform at https://imagemagick.org/script/download.php

installing is not the problem.
i have problems how to execute the task:

my goal is to automatically convert all new TIFF files that i backup on my NAS to JPG and store the JPG files in a second folder on the NAS.

**Reply by fmw42** [reply]
Q8 is only the bit-depth for images, it is not the version. You want something like

ImageMagick 6.9.11-27 Q8 or 7.0.10-27 Q8

Odd that they have Q8 and not Q16.  Q8 will be more limited in some operations and in precision, but will use less memory.

The version on the link says 7.0.7.28, which is rather old compare to the current 7.0.10.27, but should work. It might not have all the bug fixes or enhancements that have taken place since 2018-03-24 when it was released. So it is over 2 years old.

https://www.qnapclub.eu/de/search shows a Q16 version, which would be better, but uses more memory.

But you still have not identified what platform all that is on. NAS is a disk drive, not an operating system/platform.

**Comment by Gothmoth88** [comment]
it is a qnap NAS it uses QTS.

https://www.qnap.com/en/qts4/con_show.php?op=showone&cid=1

but i don´t know why that is important for my problem.  :)
my problem is more with the commandline usage of imagemagick not with getting it installed.

**Comment by fmw42** [comment]
OK. the link says it is Linux. What is your issue with the command line?  Have you read the documentation. If not, start here:


http://www.imagemagick.org/discourse-server/viewtopic.php?f=1&t=9620
http://www.imagemagick.org/script/command-line-processing.php
https://imagemagick.org/script/command-line-options.php
http://www.imagemagick.org/Usage/reference.html
http://www.imagemagick.org/Usage/
https://github.com/ImageMagick/usage-markdown
https://imagemagick.org/script/porting.php#cli
